package.json

```diff
@@ -46,7 +46,7 @@
     "browzers": "1.2.0",
     "bulk-require": "0.2.1",
     "mocha": "~1.16.2",
-    "phantomjs": "1.9.19",
+    "phantomjs-prebuilt": "^2.1.4",
```

Comment:
please see how the other dependencies are not specified like this. Please be consistent.